wiki/Contributing-to-the-Wiki.md

```diff
@@ -0,0 +1,67 @@
+`![](https://img.shields.io/static/v1?label=dnd5e&message=2.0.0&color=informational)`  
+
+# Submitting to the Wiki
+
+## Create a Wiki Issue
+Just like any other issue on the repo, the title should reflect a brief description of the task at hand, and the Description should provide us with information about what will be added to the wiki. Make sure to include all of your assets in this issue, as mentioned in the style guide above.  
+Example: Review the Issue created for submitting this document you're currently reading here: [Wiki - Create a Contributing to the Wiki guide](https://github.com/foundryvtt/dnd5e/issues/2379)
+
+## Submit a PR
+In order to contribute directly to the Wiki, you will need to fork the dnd5e repo and make a branch from the `publish-wiki` branch of the repo, then you will be able to add your Markdown files or update the existing files, and submit a Pull Request.  
+
+### Fork the `dnd5e` Repo
+To contribute to the wiki, [fork this project](https://docs.github.com/en/get-started/quickstart/fork-a-repo). Once you have forked the dnd5e repo, you will want to create a branch from the `publish-wiki` branch of the dnd5e repo. To do this, click on the branches button, then click on New Branch. Give your new Branch a descriptive name, then make sure to update the Source to the upstream repository `foundryvtt/dnd5e`and select the `publish-wiki` branch. You only need to fork the `dnd5e` repo once, but you will create a new branch for each new issue/PR.  
+
+https://github.com/foundryvtt/dnd5e/assets/86370342/26c67b33-9260-4bfa-a9be-78054a248d1c
+
+### Make Your Updates  
+Either edit the existing files, or add your own Markdown files within the `wiki` folder of your newly created branch, when done click the `Commit changes` button, you will be asked to either `Save and Merge` or `Create a New Branch`, choose `Save and Merge`. This will commit the changes to your branch, keep in mind that each commit to the branch will be recorded and listed individually in your PR, so we recommend making sure to `Preview` your changes prior to commiting to catch any minor mistakes, or editing your files in a separate editor such as Notepad ++, VSCode, or your text editor of choice, to keep the PR as clean as possible.  
+
+https://github.com/foundryvtt/dnd5e/assets/86370342/251f4dc6-9f40-4ec0-966b-e1caeadc31e8
+
+
+### Submit a PR 
+You will now be able to submit those changes back to the DnD5e system via a [pull request (PR)](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-a-pull-request) against the `publish-wiki` branch.  
+From the main page of your repo, click the `Contribute` button and `Open pull request` for the system. When submitting a PR for the wiki, make sure the base repository is `foundryvtt/dnd5e` and the base branch is `publish-wiki`. Give the PR a name, and reference your original issue in the description.
+Your PR is now submitted and will be reviewed by the Dnd5e team!
```

Comment:
Just to avoid any confusion here with the actual 'DnD5e' team:
```suggestion
Your PR is now submitted and will be reviewed by the repo maintainers!
```